templates/cronjob.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: platform
+  name: aws-credentials-updater
+
+---
+
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: platform
+  name: secret-edit
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list", "update"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: platform
+  name: aws-credentials-updater-rolebinding
+subjects:
+- kind: ServiceAccount
+  name: aws-credentials-updater
+roleRef:
+  kind: Role
+  name: secret-edit
+  apiGroup: rbac.authorization.k8s.io
+
+---
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: platform
+  name: rotate-keys
+spec:
+  schedule: "* */23 * * *" 
```

Comment:
You could add an inline comment explaining the frequency. Is it "every 23 minutes starting at minute 0" (not sure)?